Source/Dafny/Translator.cs

```diff
@@ -8486,6 +8486,10 @@ private class WFOptions
       return name;
     }
 
+    private Expr newOneHeapExpr(IToken tok) {
```

Comment:
The style in C# is to write method names with an initial capital letter. Please rename.